lib/services/services_lsb.c

```diff
@@ -261,12 +261,12 @@ services__lsb_agent_exists(const char *agent)
  * \internal
  * \brief Prepare an LSB action
  *
- * \param[in] op  Action to prepare
+ * \param[in,out] op  Action to prepare
  *
  * \return Standard Pacemaker return code
  */
 int
-services__lsb_prepare(svc_action_t *op)
+services__lsb_prepare(const svc_action_t *op)
```

Comment:
Not const. This is valid, but members of a member change.

Reply:
Yeah that's a close call. The signature doesn't have to match the doxygen documentation since they describe different things, but it is less confusing when they align.